<?xml version="1.0"?>
<launch>

    <arg name="id" default="1" />
    <arg name="pos_type" default="local" />
    <arg name="x" default="0" />
    <arg name="y" default="0" />
    <arg name="output" default="log" />

    <!-- Position provider -->
    <node pkg="area_provider" type="area_provider" name="area_provider_$(arg id)" output="$(arg output)" required="true">
        <param name="pos_type" value="$(arg pos_type)" />
        <param name="x" value="$(arg x)" />
        <param name="y" value="$(arg y)" />
        <rosparam file="$(find area_provider)/param/area_provider.yaml" command="load" />
    </node>

</launch>
